pkg/server/api/svid/v1/service.go

```diff
@@ -116,7 +116,7 @@ func (s *Service) MintX509SVID(ctx context.Context, req *svidv1.MintX509SVIDRequ
 		telemetry.SPIFFEID:  id.String(),
 		telemetry.DNSName:   strings.Join(csr.DNSNames, ","),
 		telemetry.Subject:   csr.Subject,
-		telemetry.ExpiresAt: x509SVID[0].NotAfter.Unix(),
+		telemetry.ExpiresAt: x509SVID[0].NotAfter.UTC().Unix(),
```

Comment:
Curious why the call to `UTC()` was needed. Since you are calling Unix() you will get back seconds since unix epoch, which will take the time zone into account.

Reply:
I saw the same approach in another cases, so I add it, I'll remove it